*** calendar	2010-10-20 23:00:10.993535035 -0400
--- calendar-new	2010-10-20 23:00:10.993535035 -0400
***************
*** 0 ****
--- 1,35 ----
+ <?xml version="1.0"?>
+ <contents><![CDATA[ 
+ <table border="0" cellPadding="2" cellSpacing="2" width="100%">
+ 
+ <tr>
+       <td colspan="2" bgcolor="#f0f0f0">
+       <font face="Arial, Helvetica, sans-serif" size="2">
+       <b>8:00 PM - 9:00 PM on 10/24/10:</b>
+       </font>
+       </td>
+       </tr>
+       <tr>
+       <td width="20">
+       <img SRC="/images/spacer.gif" alt="" width="20" height="8" border="0">
+       </td>
+       <td width="100%">
+       <font face="Arial, Helvetica, sans-serif" size="2">
+       <a href='/webapps/blackboard/execute/editCalendarEvent?&course_id=_6169_1&editCalendarEventAction=view&context=personal&filter=all&view=day&event_id=_2383_1'>
+       <b>Assignment #1 Doc's Due Today</b>
+       </a>
+       <br>
+ Assignment #1 Doc's Due Today</font>
+       </td>
+       </tr>
+ <tr>
+ <td colspan="2" align="RIGHT">
+ <font face="Arial, Helvetica, sans-serif" size="2">
+ <a href="/webapps/blackboard/execute/viewCalendar?context=personal&filter=all&view=month">more...</a>
+ 
+ </font>
+ </td>
+ </tr>
+ </table>
+ 
+                  ]]></contents>
\ No newline at end of file
