*** announcements	2010-11-15 15:54:28.585103001 -0500
--- announcements-new	2010-11-15 15:54:28.585103001 -0500
***************
*** 0 ****
--- 1,34 ----
+ <?xml version="1.0"?>
+ <contents><![CDATA[ <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+ <table border="0" cellPadding="2" cellSpacing="2" width="100%">
+ 
+ 		<tr>
+ 			<td><font face="Arial, Helvetica, sans-serif" size="2"> No Institution Announcements have been posted in the last 7 days.</font></td>
+ 		</tr>
+ 		<tr>
+ <td bgcolor="#f0f0f0">
+ <font face="Arial, Helvetica, sans-serif" size="2">
+ <b>
+ 10F_CST8165_010 Communications and Networks II
+ </b>
+ </font>
+ </td>
+ </tr>
+ <tr><td colspan='2'><font face="Arial, Helvetica, sans-serif" size="2"><li><a href="/webapps/blackboard/execute/announcement?method=search&context=mybb&searchSelect=_6169_1">Assignment #2 - Local SMTP Server</a></li></font></td></tr><tr>
+ <td bgcolor="#f0f0f0">
+ <font face="Arial, Helvetica, sans-serif" size="2">
+ <b>
+ 10F_CST8261_010 Security, Encryption
+ </b>
+ </font>
+ </td>
+ </tr>
+ <tr><td colspan='2'><font face="Arial, Helvetica, sans-serif" size="2"><li><a href="/webapps/blackboard/execute/announcement?method=search&context=mybb&searchSelect=_6200_1">Test 2</a></li></font></td></tr><tr>
+ 			<td align="RIGHT"><font face="Arial, Helvetica, sans-serif" size="2">
+ 		    <a
+ 				href="/webapps/blackboard/execute/announcement?method=search&context=mybb&viewChoice=3">more...</a>...
+ 				</font></td>
+ 		</tr>
+ 		</table>
+ 
+                  ]]></contents>
\ No newline at end of file
