*** calendar	2010-11-15 16:13:13.412542001 -0500
--- calendar-new	2010-11-21 19:21:08.074502000 -0500
***************
*** 5,11 ****
  <tr>
        <td colspan="2" bgcolor="#f0f0f0">
        <font face="Arial, Helvetica, sans-serif" size="2">
!       <b>4:00 PM - 6:00 PM on 11/15/10:</b>
        </font>
        </td>
        </tr>
--- 5,11 ----
  <tr>
        <td colspan="2" bgcolor="#f0f0f0">
        <font face="Arial, Helvetica, sans-serif" size="2">
!       <b>4:00 PM - 5:00 PM on 11/21/10:</b>
        </font>
        </td>
        </tr>
***************
*** 15,22 ****
        </td>
        <td width="100%">
        <font face="Arial, Helvetica, sans-serif" size="2">
!       <a href='/webapps/blackboard/execute/editCalendarEvent?&course_id=_6169_1&editCalendarEventAction=view&context=personal&filter=all&view=day&event_id=_3826_1'>
!       <b>Assignment #2 Demo Due Today</b>
        </a>
        <br>
  </font>
--- 15,22 ----
        </td>
        <td width="100%">
        <font face="Arial, Helvetica, sans-serif" size="2">
!       <a href='/webapps/blackboard/execute/editCalendarEvent?&course_id=_6169_1&editCalendarEventAction=view&context=personal&filter=all&view=day&event_id=_4332_1'>
!       <b>Assignment #2 Docs Due Today</b>
        </a>
        <br>
  </font>
***************
*** 25,31 ****
  <tr>
        <td colspan="2" bgcolor="#f0f0f0">
        <font face="Arial, Helvetica, sans-serif" size="2">
!       <b>4:00 PM - 5:00 PM on 11/21/10:</b>
        </font>
        </td>
        </tr>
--- 25,31 ----
  <tr>
        <td colspan="2" bgcolor="#f0f0f0">
        <font face="Arial, Helvetica, sans-serif" size="2">
!       <b>4:00 PM - 6:00 PM on 11/22/10:</b>
        </font>
        </td>
        </tr>
***************
*** 35,42 ****
        </td>
        <td width="100%">
        <font face="Arial, Helvetica, sans-serif" size="2">
!       <a href='/webapps/blackboard/execute/editCalendarEvent?&course_id=_6169_1&editCalendarEventAction=view&context=personal&filter=all&view=day&event_id=_4332_1'>
!       <b>Assignment #2 Docs Due Today</b>
        </a>
        <br>
  </font>
--- 35,42 ----
        </td>
        <td width="100%">
        <font face="Arial, Helvetica, sans-serif" size="2">
!       <a href='/webapps/blackboard/execute/editCalendarEvent?&course_id=_6169_1&editCalendarEventAction=view&context=personal&filter=all&view=day&event_id=_4329_1'>
!       <b>Lab 4 - Demo Due Today</b>
        </a>
        <br>
  </font>
***************
*** 45,51 ****
  <tr>
        <td colspan="2" bgcolor="#f0f0f0">
        <font face="Arial, Helvetica, sans-serif" size="2">
!       <b>4:00 PM - 6:00 PM on 11/22/10:</b>
        </font>
        </td>
        </tr>
--- 45,51 ----
  <tr>
        <td colspan="2" bgcolor="#f0f0f0">
        <font face="Arial, Helvetica, sans-serif" size="2">
!       <b>4:00 PM - 5:00 PM on 11/28/10:</b>
        </font>
        </td>
        </tr>
***************
*** 55,62 ****
        </td>
        <td width="100%">
        <font face="Arial, Helvetica, sans-serif" size="2">
!       <a href='/webapps/blackboard/execute/editCalendarEvent?&course_id=_6169_1&editCalendarEventAction=view&context=personal&filter=all&view=day&event_id=_4329_1'>
!       <b>Lab 4 - Demo Due Today</b>
        </a>
        <br>
  </font>
--- 55,62 ----
        </td>
        <td width="100%">
        <font face="Arial, Helvetica, sans-serif" size="2">
!       <a href='/webapps/blackboard/execute/editCalendarEvent?&course_id=_6169_1&editCalendarEventAction=view&context=personal&filter=all&view=day&event_id=_4333_1'>
!       <b>Lab 4 Doc's Due Today</b>
        </a>
        <br>
  </font>
